lib/components/Tooltip.tsx

```diff
@@ -17,154 +12,17 @@ type Props = {
-  static currentHoveredElement: Element | undefined;
-  static virtualElement: VirtualElement = {
-    getBoundingClientRect: () =>
-      Tooltip.currentHoveredElement?.getBoundingClientRect() ?? NULL_RECT,
-  };
-
-  getDOMNode() {
-    // HACK: We don't want to create a wrapper, as it could break the layout
-    // of consumers, so we use findDOMNode.
-    // This is usually bad as refs are usually better, but refs did
-    // not work in this case, as they weren't propagating correctly.
-    // A previous attempt was made as a render prop that passed an ID,
-    // but this made consuming use too unwieldly.
-    // Because this component is written in TypeScript, we will know
-    // immediately if this internal variable is removed.
-    //
-
-    return findDOMNode(this) as Element;
-  }
-
-  componentDidMount() {
-    const domNode = this.getDOMNode();
-
-    if (!domNode) {
-      return;
-    }
-
-    domNode.addEventListener('mouseenter', () => {
-      let renderedTooltip = Tooltip.renderedTooltip;
-      if (renderedTooltip === undefined) {
-        renderedTooltip = document.createElement('div');
-        renderedTooltip.className = 'Tooltip';
-        document.body.appendChild(renderedTooltip);
-        Tooltip.renderedTooltip = renderedTooltip;
-      }
-
-      Tooltip.currentHoveredElement = domNode;
-
-      renderedTooltip.style.opacity = '1';
-
-      this.renderPopperContent();
-    });
-
-    domNode.addEventListener('mouseleave', () => {
-      this.fadeOut();
-    });
-  }
-
-  fadeOut() {
-    if (Tooltip.currentHoveredElement !== this.getDOMNode()) {
-      return;
-    }
-
-    Tooltip.currentHoveredElement = undefined;
-    Tooltip.renderedTooltip!.style.opacity = '0';
-  }
-
-  renderPopperContent() {
-    const renderedTooltip = Tooltip.renderedTooltip;
-    if (!renderedTooltip) {
-      return;
-    }
-
-    render(<span>{this.props.content}</span>, renderedTooltip, () => {
-      let singletonPopper = Tooltip.singletonPopper;
-      if (singletonPopper === undefined) {
-        singletonPopper = createPopper(
-          Tooltip.virtualElement,
-          renderedTooltip!,
-          {
-            ...DEFAULT_OPTIONS,
-            placement: this.props.position || 'auto',
-          },
-        );
-
-        Tooltip.singletonPopper = singletonPopper;
-      } else {
-        singletonPopper.setOptions({
-          ...DEFAULT_OPTIONS,
-          placement: this.props.position || 'auto',
-        });
-
-        singletonPopper.update();
-      }
-    });
-  }
-
-  componentDidUpdate() {
-    if (Tooltip.currentHoveredElement !== this.getDOMNode()) {
-      return;
-    }
-
-    this.renderPopperContent();
-  }
-
-  componentWillUnmount() {
-    this.fadeOut();
-  }
-
-  render() {
-    return this.props.children;
-  }
+export function Tooltip(props: Props) {
+  const { content, children, position } = props;
+
+  return (
+    <Floating
+      childrenNoWrap
+      content={<div className="Tooltip">{content}</div>}
```

Comment:
```suggestion
      content={content}
      contentClasses="Tooltip"
```
You can use contentClasses instead second wrapping